src/Config/ConfigurationStore.php

```diff
@@ -0,0 +1,168 @@
+<?php
+
+namespace Eppo\Config;
+
+use Eppo\Bandits\BanditVariationIndexer;
+use Eppo\Cache\CacheType;
+use Eppo\Cache\NamespaceCache;
+use Eppo\DTO\Bandit\Bandit;
+use Eppo\DTO\Flag;
+use Eppo\Exception\InvalidConfigurationException;
+use Psr\SimpleCache\CacheInterface;
+use Psr\SimpleCache\InvalidArgumentException;
+
+class ConfigurationStore implements IConfigurationStore
```

Comment:
Similar story with this file; actual change is `getBandit` method and additional cache references for the Bandit cache.